Automattic/wp-calypso
# Staging sites: optimize hosting config loading card

- Fixes https://github.com/Automattic/dotcom-forge/issues/1943

## Proposed Changes

* Make a first call to fetch the transfer status, which makes the card load faster.
* Avoid showing the transfer progress bar if the transfer is not initiated.

## Testing Instructions



* Create a business site and transfer it to atomic. This is the parent site.
* Go to Settings > Hosting configuration
* Observe the Staging sites card displays the "Create staging site" button.
* Click on it
* Observe the transfer progress bar appears. Wait for a few minutes.
* Observe the staging site info appears and also the button to `Manage staging site`
* Refresh the page
* Observe the staging site info that appears.

## Pre-merge Checklist
